rfi_file_monitor/application.py

```diff
@@ -173,6 +185,18 @@ def do_startup(self):
         # add our help window, which will be shared by all appwindows
         self._help_window = HelpWindow(self._pango_docs_map)
 
+        # acquire google analytics context
+        self._google_analytics_context = GoogleAnalyticsContext(
+            endpoint="https://www.google-analytics.com/collect",
+            tracking_id="UA-184737687-1",
```

Comment:
should this part be in a config? Or are we happy to leave this in source...

Reply:
nope source is fine, this tracking-id is unique for us. You will see that any website that uses GA tracking has such an id hardcoded as well.